src/main/java/de/tum/in/www1/artemis/config/migration/MigrationEntry.java

```diff
@@ -0,0 +1,19 @@
+package de.tum.in.www1.artemis.config.migration;
+
+import java.io.Serializable;
+
+public abstract class MigrationEntry implements Serializable {
+
+    public abstract void execute();
+
+    /**
+     * @return Author of the entry. Either full name or GitHub name.
+     */
+    public abstract String author();
+
+    /**
+     * Format YYYYMMDD_HHmmss
```

Comment:
I just thought about the following:
Are time zones critically here?

I don't think so but just to be sure :)